docs/index.md

```diff
@@ -17,40 +17,40 @@ Islandora:
 - **Offers flexibility** - As Islandora content is Drupal content, migrations and batch editing can be done through Drupal's built-in migrate framework and controlled vocabularies created using Drupal taxonomies. Contributed Drupal modules such as [Solr Search API](https://www.drupal.org/project/search_api_solr) enable in-site search, and [Matomo Analytics](https://www.drupal.org/project/matomo) provides usage metrics for site analytics.
 - **Is a community** - A [dedicated, active community of users and developers](https://groups.google.com/forum/#!forum/islandora) is working to push new features, collaborate on improvements, design custom solutions, and create extended functionality. Some of these for Islandora 8 take the form of [Recipes](https://github.com/Islandora-Labs/Islandora-Cookbook).
 
-## Demo
+## Try Islandora
 
-Visit [http://future.islandora.ca/](http://future.islandora.ca/).
+### Sandbox
 
-## Quickstart
+To try Islandora without installing anything, visit [http://future.islandora.ca/](http://future.islandora.ca/). Anyone can log in to this sandbox as an administrator (credentials are below and on the front page) and explore the interface! However, this site is refreshed periodically so your changes will not be permanent. This site uses Islandora Defaults, a way of setting up Islandora for demonstration purposes - this is not the only way that Islandora can be made to work!
```

Comment:
Add "This sandbox also demonstrates some Drupal contrib modules that are not standard in Islandora, such as View Bulk Edit, which powers the Batch Edit Page."